src/Resources/config/config.xml

```diff
@@ -353,6 +353,20 @@
             <label lang="de-DE">Bestandskunden, die noch nicht geprüft wurden, werden bei einem Login und Aufruf einer Adressformular-Seite automatisch geprüft.</label>
             <defaultValue>false</defaultValue>
         </input-field>
+
+        <input-field type="bool">
```

Comment:
Can you create separate cards for email check and name check (person check), similarly how you did it for phone check? Otherwise it would be a bit misleading to find them in address-management service area.